src/setupTests.js

```diff
@@ -23,6 +24,8 @@ global.markdownitBr = require('markdown-it-br')
 global.markdownitMath = require('markdown-it-synapse-math')
 global.ResizeObserver = ResizeObserver
 
+setupIntersectionMocking(jest.fn)
```

Comment:
Not sure what this does exactly, but it gets rid of test warnings telling us to do this.